src/utils.js

```diff
@@ -1,13 +1,5 @@
 export function toParams(query) {
-  const q = query.replace(/^\??\//, '');
```

Comment:
Since this regex is apparently no longer needed I think we can remove the same regex replacement that is done where `toParams` is called?
https://github.com/okteto/react-oauth2-login/blob/be6a810f7509ed44621f8794f459ec4a4863c710/src/LoginOAuth2Window.js#L39

i.e.
```js
const params = toParams(popup.location.search); 
```

BTW, this regex was removing optional `?` (parent regex removal before would explain the `\??`) and an initial `/` for some supposed good reason. I suspect `URLSearchParams` will interpret something like `?/foo=bar` as `{'/foo': 'bar'}`, though. can you add a test-case what `toParams` so we have prev behavior covered and  don't cause an accidental regression?

Reply:
Good catch. I didn't realize about the `/` char. Is there any reason to explicitly handle this particular case @rlamana?

Reply:
Actually, this regex is only capturing the initial `?` if it is followed by a `/`. Otherwise, the question mark won't be removed. Seems like an accidental error from the past.

Reply:
@bgonp no, since `?` is optional it would also match leading `/`, see:
<img width="276" alt="Screenshot 2023-03-01 at 09 49 44" src="https://user-images.githubusercontent.com/978461/222089619-23fc307b-f17f-498d-890c-297ab7fb35ec.png">

Reply:
This is a very sensitive change. Make sure we grow the test coverage for this particular function with as many cases as possible to avoid causing any regressions.

Reply:
What I mean is that using that regex, if the question mark is not followed by a `/`, then the `?` won't be removed.
![image](https://user-images.githubusercontent.com/13395148/222095659-9fc83279-db7c-47ae-9357-4d1afaf78123.png)
This malformed regex could be the original motivation to add this in the first place:
https://github.com/okteto/react-oauth2-login/blob/be6a810f7509ed44621f8794f459ec4a4863c710/src/LoginOAuth2Window.js#L39

Reply:
Previously we were applying `replace(/^\?/, '')` and then `replace(/^\??\//, '')` so we were removing `?`, `/` and `?/` at the beginning of the string.
Since we are using [`window.location.search`](https://developer.mozilla.org/en-US/docs/Web/API/Location/search) which doesn't include the `/` I think it is safe to rely on `URLSearchParams` to parse the query params, since it will remove the `?` when parsing values.